cli/src/cmd/run.rs

```diff
@@ -149,14 +165,14 @@ impl Cmd for RunArgs {
                     if evm_opts.verbosity > 4 || !result.success {
                         // print setup calls as well
                         traces.iter().for_each(|trace| {
-                            trace.pretty_print(0, &known_contracts, &mut ident, runner.evm, "");
+                            trace.pretty_print(0, &known_contracts, &mut ident, &vm(), "");
```

Comment:
`pretty_print` likely needs refactoring anyway, and since we always call `update_identified` inside a test, we no longer need the vm

Reply:
likely this should be done in a separate PR. `pretty_print` should just build up a string that can be printed in one shot